nevergrad/benchmark/optgroups.py

```diff
@@ -37,6 +37,10 @@ def get_optimizers(*names: str, seed: tp.Optional[int] = None) -> tp.List[Optim]
     return optims
 
 
+def get_str_optimizers(*names: str, seed: tp.Optional[int] = None) -> tp.List[str]:
+    return [optim.name if not isinstance(optim, str) else optim for optim in get_optimizers(*names, seed=seed)]
```

Comment:
Why do you use this and not `get_optimizers("splitters")`?
This should *not* be needed because it will not always work.